examples/simple-form/index.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import { Formik, Form } from 'formik';
+import {
+  TextField,
+  DebuggerView,
+} from '@department-of-veterans-affairs/formulate';
+
+import { defineCustomElements } from 'web-components/loader';
+
+void defineCustomElements();
```

Comment:
The `void` is new to me. What's this doing?

Reply:
`eslint-typescript` was grumpy that I wasn't doing anything with the promise that `defineCustomElements()` returns, but I don't actually want to do anything, so `void` got rid of that error. To be honest, I don't know why, but that's what the error message told me to do if I wasn't going to use the returned promise.

I'll look it up. :detective:

Reply:
[MDN's documentation on the `void` operator.](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/void)

Looks like it's a "use it or be intentional about ignoring it" kind of a linting rule I ran into. :shrug:

Reply:
Good to know,

Also, since we have a `simple-form` (and will presumably have more complicated ones in the near future), do you think it makes sense to move the `web-components` imports & `defineCustomElements()` call out of this file and into `examples/index.jsx`?

Reply:
I have it set up right now intentionally to have each example be totally standalone. Moving that up into `examples/index.jsx` would do nothing for `simple-form/index.jsx`.

It's probably _better_ this way to avoid any strangeness with a shared setup, but also getting each example to work with a template or whatever is frankly just more work. :stuck_out_tongue: 

I thought about making all examples as part of a SPA, but later on, when we start working on routing, we'd then have like SPAs inside of the overall SPA, and that gets...funky, potentially. So I figured it's a little extra copy pasta this way, but each example is nice and encapsulated. Easy to debug later if we need to since there's very little magic under the hood.

Reply:
What we _could_ do is move it into a `<script>` in the `template.html`, but then we'd have to add another entry file to the webpack config to emit the web components. :thinking: 

I'll leave that for another PR, though, if we want to go that route.